airflow/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -210,13 +210,19 @@ def get_conn(self) -> BlobServiceClient:
             if sas_token.startswith("https"):
                 return BlobServiceClient(account_url=sas_token, **extra)
             else:
+                if not account_url.startswith("https://"):
```

Comment:
Should we do this up after we initially set `account_url`? Then we only need to do it once?

Reply:
Ah I see, so I think we could do it before this if block instead then. But if you want to keep it twice, I'm good either way.